src/AksArc/AksArc.Autorest/test/AzAksArcCluster.Tests.ps1

```diff
@@ -67,7 +67,7 @@ Describe 'AzAksArc' {
         } | Should -Not -Throw
     }
 
-    It 'CreateNodepool' {
+    It 'CreateNodepool' -skip {
```

Comment:
Why is this skipped? Should there be new cases for the added functionality?

Reply:
@NoriZC I talked to your team and they said that these methods don't work with the test engine since they are using an Invoke-AzRestMethod function and that I should mark them with the -skip flag. 

I have ran these methods locally and they work and we also are adding tests in our own pipelines to validate more scenarios.